redash/query_runner/snowflake.py

```diff
@@ -96,14 +104,26 @@ def _get_connection(self):
             else:
                 host = "{}.snowflakecomputing.com".format(account)
 
-        connection = snowflake.connector.connect(
-            user=self.configuration["user"],
-            password=self.configuration["password"],
-            account=account,
-            region=region,
-            host=host,
-            application="Redash/{} (Snowflake)".format(__version__.split("-")[0]),
-        )
+        params = {
+            "user": self.configuration["user"],
+            "account": account,
+            "region": region,
+            "host": host,
+            "application": "Redash/{} (Snowflake)".format(__version__.split("-")[0])
+        }
+
+        if self.configuration.__contains__("password"):
+            params["password"] = self.configuration["password"]
+        elif self.configuration.__contains__("private_key_b64") or self.configuration.__contains__("private_key_File"):
+            private_key_b64 = self.configuration.get("private_key_b64") or self.configuration.get("private_key_File")
+            private_key_bytes = b64decode(private_key_b64)
+            private_key_pwd = self.configuration.get("private_key_pwd")
```

Comment:
I think private_key_pwd need to be None if private_key_pwd is empty string.

Reply:
self.configuration.get("private_key_pwd")

https://github.com/getredash/redash/pull/7371#discussion_r2245661574

I just think this line can be like:
```
private_key_pwd = self.configuration.get("private_key_pwd") or None
```